src/components/lazy-load-css.tsx

```diff
@@ -0,0 +1,25 @@
+import React from "react"
+import { Helmet } from "react-helmet"
+
+// Technique copied from https://web.dev/defer-non-critical-css/#optimize
+
+interface Props {
+    url: string
+}
+
+const LazyLoadCss: React.FC<Props> = ({ url }) => {
+    return (
+        <Helmet>
+            <link
+                rel="preload"
+                href={url}
+                as="style"
+                // @ts-ignore
+                onload="this.onload=null;this.rel='stylesheet'" // eslint-disable-line react/no-unknown-property
+            />
+            <noscript>{`<link rel="stylesheet" href="${url}" />`}</noscript>
+        </Helmet>
+    )
+}
```

Comment:
nit: don't need braces and return statement

Reply:
I never do an implicit return as I always end up needing to declare some variables at some point down the line and then you get an annoying indentation diff so i always just do explicit return in react components 🙂